test/test_pelagos.py

```diff
@@ -11,11 +11,13 @@
 
 import network_manager
 import pelagos
+import hw_node
 import pxelinux_cfg
 
 logging.basicConfig(format='%(asctime)s | %(name)s | %(message)s',
                     level=logging.DEBUG)
 test_pxelinux_cfg_root_dir = '/tmp/tftp'
+test_conman_dir = '/tmp/conman'
```

Comment:
This must be a randomized tempdir dir for thread safety

Reply:
there is many the same mistake, it should be done separately for fixing all, ticket created
https://jira.suse.com/browse/SES-1532